components/card.css

```diff
@@ -0,0 +1,81 @@
+.card-container{
+    width: 22rem;
+    /* border: 2px solid red; */
+    display: flex;
+    flex-direction: column;
+    gap: 1rem;
+    box-shadow: 0px 0px 10px 4px rgb(0 0 0 / 5%);
+    padding: 1rem;
+}
+.card-image{
+    
+}
+.card-title{
+    font-size: 1.5rem;
+    font-weight: normal;
+}
+.card-description{
+    font-size: 1rem;
+    color: var(--color-font);
+}
+.card-btn-container{
+    display: flex;
+    justify-content: flex-start;
+    gap: 1rem;
+}
+.card-btn{
+    /* border-radius: 0.5rem; */
+    border: none;
+    cursor: pointer;
+    padding: 0.5rem;
+    background-color: white;
+}
+.card-btn-primary{
+    background-color: var(--color-primary);
+    color: var(--color-white);
+}
+.card-btn-primary:hover{
+    opacity: 0.9;
+}
+.card-btn-secondary{
+    margin-right: auto;
+}
+.card-btn-secondary:hover{
+    color: var(--color-font);
+    background-color: var(--color-light-gray);
+}
+.card-btn-icon{
+    border-radius: 50%;
+    padding: 0.5rem;
+}
+.card-btn-icon:hover{
+    background-color: var(--color-light-gray)
+}
+.card-badge{
+    position: absolute;
+    color: var(--color-white);
+    background-color: var(--color-primary);
+    padding: 4px;
+    border-radius: 0px 0px 4px 0px;
+}
+.card-overlay-container{
+    opacity: 0.7;
+    display: flex;
+    justify-content: center;
+}
+.card-overlay-text{
+    position: absolute;
+    color: black;
+    background-color: var(--color-background);
+    padding: 0.5rem;
+    opacity: 0.9;
+    align-self: center;
+    font-size: 2rem;
+}
+.card-horizontal{
+    flex-direction: row;
+    width: fit-content;
+}
+.card-container .card-container{
+    box-shadow: none;
```

Comment:
What's the use of this? Your card already has a border so why are you setting it to none.

Reply:
This is used to remove shadow effect in the container inside the card.